### PR TITLE
92 backend: endpoints

### PR DESCRIPTION
**Pull request by timabloom**
I added authorization for certain transaction endpoints by using authorId so that only the author of the project can accept deny kick and make an invite

### CODE REVIEW

**Comment by timabloom** [comment]
I am using this branch to finish up my work on the database. I have implemented for both projects and users to return amount of skills sought after in descending order

File: Backend/Controllers/ProjectsController.cs
Comment (Oshulten) [inline Backend/Controllers/ProjectsController.cs:1]: I think that its superfluous having an endpoint name like 

GetProjectByProjectId/{projectId}

The most import thing to see in the url (for the frontend and for checking swagger) is
- The return type (in this case ProjectResponse)
- The path and and query arguments (in this case projectId)

I would suggest a url like

GetProjectResponse/{projectId}

Comment (Oshulten) [inline Backend/Controllers/ProjectsController.cs:1]: Same thing here, adding byProjectIdbyAuthorId... doesn't really clarify, it just makes the url longer

File: Backend/Controllers/UsersController.cs
Comment (Oshulten) [inline Backend/Controllers/UsersController.cs:1]: Usually when giving paginated results, you specify in the response whether there's more data to fetch be returning a nextPage = 3, or nextPage = null if there are no more pages of data to fetch

File: Backend/Controllers/ProjectsController.cs
Comment (timabloom) [reply Backend/Controllers/ProjectsController.cs:1]: I agree

File: Backend/Controllers/UsersController.cs
Comment (timabloom) [reply Backend/Controllers/UsersController.cs:1]: I will add this to to the response

**Pull request closed without merging**